tests/Fixtures/inc/functions/rocketFindWpconfigPath.php

```diff
@@ -0,0 +1,23 @@
+<?php
+return [
+	'vfs_dir' => 'public/',
+	'structure' => [
+		'wp-config.php' => 'test content here',
```

Comment:
I’d suggest creating a `wordpress`  directory:

```php 
‘wordpress’ => [
	'wp-config.php' => 'test content here',
],
```

Why? The alt version will check on level up via `dirname()`. Using this structure, we are simulating the Wordpress code inside of a sub directory with the alt version outside of it.